source/renderer/app/stores/WalletsStore.js

```diff
@@ -59,6 +59,7 @@ export default class WalletsStore extends Store {
   // REQUESTS
   @observable active: ?Wallet = null;
   @observable activeValue: ?BigNumber = null;
+  @observable hardwareActive: ?Wallet = null;
```

Comment:
@DeeJayElly you can remove this. It is defined but never used

Reply:
Fixed